Documents/Coding Standards/CodingStandards.tex

```diff
@@ -34,5 +34,33 @@
   \subsection{Swift}
   For Swift we will use \href{https://swift.org/documentation/api-design-guidelines/}{Apple design guidelines}.
   
+  \section{Git Structure}
+  \subsection{Branching}
+  Rule one, always, is never work out of \texttt{master}. A new branch is created when someone wants to implement, or experiment with something that is not theirs, or in the \texttt{master}, so they will branch off of the relevant branch, to make their proposed changes.
```

Comment:
The always makes reading the sentence awkward, maybe consider removing it?

Reply:
I'm not saying that the meaning of always shouldn't be there. Just that it's placement in the sentence structure is awkward to read. For example, something like "Rule one, never work out of master. Ever."

Reply:
I got what you meant. It's just that I disagree, I think it makes it clear that the first of working with GitHub is to never work out master, so it always applies.

Reply:
Ultimately, I think we need a third party to decide.

Reply:
Okay, then I guess we need other reviewers. I'll notify the other members that they should participate in PR reviews.